client-core/src/wallet/syncer.rs

```diff
@@ -389,7 +386,7 @@ impl<'a, S: SecureStorage, C: Client, D: TxDecryptor> WalletSyncerImpl<'a, S, C,
     fn rollback_pending_tx(&mut self, current_block_height: u64) -> Result<()> {
         let mut memento = WalletStateMemento::default();
         let state =
-            service::load_wallet_state(&self.env.storage, &self.env.name, &self.env.passphrase)?
+            service::load_wallet_state(&self.env.storage, &self.env.name, &self.env.enckey)?
```

Comment:
I don't think this is a good pattern to have. Ideally we should use Service objects and their functions. It's ok for now.

Reply:
we can discuss this some time.